deadline/deadline.go

```diff
@@ -0,0 +1,120 @@
+package deadline
+
+import "github.com/filecoin-project/go-state-types/abi"
+
+// Deadline calculations with respect to a current epoch.
+// "Deadline" refers to the window during which proofs may be submitted.
+// Windows are non-overlapping ranges [Open, Close), but the challenge epoch for a window occurs before
+// the window opens.
+// The current epoch may not necessarily lie within the deadline or proving period represented here.
+type DeadlineInfo struct {
```

Comment:
In package `deadline` this should be `Info` for `deadline.Info`. Deadline might be an unfortunate package name though, liable to clash with var names.

Reply:
going with `dline.Info`